runners/maven-plugin/src/main/kotlin/DokkaMojo.kt

```diff
@@ -332,6 +332,9 @@ class DokkaMojo : AbstractDokkaMojo() {
 class DokkaJavadocMojo : AbstractDokkaMojo() {
```

Comment:
Could you please add a MavenIntegrationTest to cover this? I didn't even know that `DokkaJavadocMojo` exists 🙈

Reply:
I can try, but I have never ever done integration tests in my life before, just know theory. Can you point me to example I could base on?